CMakeLists.txt

```diff
@@ -1,7 +1,10 @@
 
-cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
 
-project(prometheus-cpp VERSION 0.9.0)
+project(prometheus-cpp
+  VERSION 0.9.1
```

Comment:
This is a pretty band-aid solution for potential conflicts with the original `prometheus-cpp`. 
Since it's still pretty early on, I suggest changing the project name to `avidbots-prometheus-cpp` and incrementing to version 1.0. This would require updating any packages that depend on it.
Especially since I suspect our code would not work with the original `prometheus-cpp` package.